### PR TITLE
CAM-11281: feat(db): provide testcontainers wrapper for camunda

### PR DESCRIPTION
**Pull request by koevskinikola**
This PR creates a Testcontainers wrapper that enables us to use a private Docker registry for spinning up Docker containers of Camunda-supported databases.

Currently, I added support for PostgreSQL, CockroachDB, MySQL, MariaDB and MS-SQL.

Related to CAM-11281

### CODE REVIEW

File: testing/README.md [hunk 1]
```diff
@@ -20,3 +20,17 @@ public void testOverrideWithAnnotation() {
 
 }
 ```
+
+## Usage of the Testcontainers wrapper
+
+1. Add a `testcontainers.property` file to the root of your test resources directory ([example file](./testing/src/test/resources/testcontainers.properties));
```
Comment (tasso94) [inline testing/README.md:26]:
```suggestion
1. Add a `testcontainers.properties` file to the root of your test resources directory ([example file](./testing/src/test/resources/testcontainers.properties));
```

File: testing/src/main/java/org/camunda/commons/testing/testcontainers/TestcontainersHelper.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.testcontainers.shaded.org.apache.commons.lang.StringUtils;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+public class TestcontainersHelper {
+
+  public static String resolveImageName(String imageProperty, String defaultImage) {
+    String image = TestcontainersConfiguration.getInstance().getEnvVarOrProperty(imageProperty, defaultImage);
+    if (image == null) {
+      throw new IllegalStateException("An image to use for Oracle containers must be configured. " +
```
Comment (tasso94) [inline testing/src/main/java/org/camunda/commons/testing/testcontainers/TestcontainersHelper.java:28]: This exception might not only occur for **Oracle** containers, right?

**Review by tasso94 — CHANGES_REQUESTED** [review]

**Comment by koevskinikola** [comment]
I'm closing this PR, as the code was migrated to the `camunda-bpm-platform` repository with https://github.com/camunda/camunda-bpm-platform/pull/1359. The review hints will be implemented there.

File: testing/src/main/java/org/camunda/commons/testing/testcontainers/TestcontainersHelper.java (hunk 2 above)
Comment (koevskinikola) [reply testing/src/main/java/org/camunda/commons/testing/testcontainers/TestcontainersHelper.java:28]: Yes, this was with the initial implementation. I will rephrase it to be more general so that it applies to all containers.
